components/sup/src/util/users.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[cfg(any(target_os="linux", target_os="macos"))]
+extern crate users;
+
+use error::{Result, Error};
+use hcore::package::PackageInstall;
+
+static LOGKEY: &'static str = "USERS";
```

Comment:
The purpose of the two character log key isn't for informing users or about their readability, but developer access and debugging. We are supposed to be using these to identify which module a log even is coming from without needing to specify the full path plus crate name of the module in the log output. This is much like the way we log error messages in the servers with a very similar format.

I would stick with two or three character log prefixes, expanding them is going to clutter the log and I don't think it makes things any easier to track down.